packages/nimble-components/src/patterns/error/styles.ts

```diff
@@ -29,17 +28,13 @@ export const styles = css`
         color: ${failColor};
         width: 100%;
         position: absolute;
-        top: ${controlHeight};
+        bottom: calc(-1 * ${errorTextFontLineHeight});
```

Comment:
This is the first component to use the shared error style that isn't a fixed `controlHeight` tall. Switch to setting `bottom` instead of `top` to handle a variable control height.